jujugui/static/gui/src/app/templates/storefront-search-row.partial

```diff
@@ -0,0 +1,113 @@
+    {{/compare}}
+    {{displayName}}
+    {{#if commercial}}
+      <span class="commercial-flag"></span>
+    {{/if}}
+  </h3>
+  {{#if tags}}
+    <ul class="tag-list">
+      {{#each tags}}
+        <li class="tag-list--item"><a href="/q?tags={{this}}">{{this}}</a></li>
+      {{/each}}
+    </ul>
+  {{/if}}
+</div>
+<div class="{{#if seriesWidth}}{{seriesWidth}}{{else}}two-col{{/if}} series__column">
+  {{#if series}}
+    <ul class="tag-list tag-list--spaced">
+      {{#each series}}
+        <li class="tag-list--item"><a href="{{url}}">{{name}}</a></li>
+      {{/each}}
+    </ul>
+  {{else}}
+    {{!Add some content so the row is spaced correctly.}}
+    &nbsp;
+  {{/if}}
+</div>
+<div class="{{#if iconsWidth}}{{iconsWidth}}{{else}}three-col{{/if}} charm-logos__column list-block__column">
+  <ul class="list-icons clearfix">
+    {{#each services}}
+      <li class="list-icons__item">
+        <a href="{{charm.url}}">
+          {{#compare 1 20 operator='>'}}
+            <p>
+              <span class="tip">
+                {{#if charm.icon}}
+                  <img class="icon-scroll list-icons__image"
+                       data-icon="{{ charm.icon }}"
+                       src="{{../STATIC_URL}}img/icons/default-charm.svg"
+                       alt="{{charm.displayName}} icon" />
+                {{else}}
+                  <img class="list-icons__image"
+                       src="{{../STATIC_URL}}img/icons/default-charm.svg"
+                       alt="{{charm.displayName}} icon" />
+                {{/if}}
+                <span class="tip-content" data-tooltip="{{ charm.display_name }}"></span>
+              </span>
+            </p>
+          {{else}}
+            <p>
+              <span class="tip">
+                <img src="{{#if charm.icon}}{{charm.icon}}{{else}}{{../STATIC_URL}}img/icons/default-charm.svg{{/if}}"
+                     class="list-icons__image"
+                     alt="{{charm.displayName}} icon" />
+                <span class="tip-content" data-tooltip="{{charm.displayName}}"></span>
+              </span>
+            </p>
+          {{/compare}}
+        </a>
+      </li>
+    {{else}}
+      <li class="list-icons__item">
+        <a href="{{ url }}">
+          {{#compare 1 20 operator='>'}}
+            <p>
+              <span class="tip">
+                {{#if icon}}
+                <img class="icon-scroll list-icons__image"
+                     data-icon="{{icon}}"
+                     src="{{../STATIC_URL}}img/icons/default-charm.svg"
+                     alt="{{displayName}} icon" />
+                {{else}}
+                <img src="{{../STATIC_URL}}img/icons/default-charm.svg"
+                     class="list-icons__image"
+                     alt="{{displayName}} icon" />
+                {{/if}}
+                <span class="tip-content" data-tooltip="{{displayName}}"></span>
```

Comment:
Just some variables that were missed in porting. In actuality, they do refer to two different things: in the first logic path (e.g., looping through services within a bundle) `displayName` is scoped to the containing loop. In the second logic path (no services, so a charm) `displayName` refers to the current search result. This is one of those places where I feel like Handlebars is actually less clear because it is less explicit. You have to be much more cognizant of variable scopes.